core/gather/driver/wait-for-condition.js

```diff
@@ -487,6 +487,18 @@ async function waitForFullyLoaded(session, networkMonitor, options) {
         throw new LighthouseError(LighthouseError.errors.PAGE_HUNG);
       }
 
+      // Log remaining inflight requests if any.
+      const inflightRequestsUrl = networkMonitor
+        .getInflightRequests()
+        .map((request) => request.url);
+      if (inflightRequestsUrl.length > 0) {
+        log.warn(
+          "waitFor",
+          "Remaining inflight requests URLs",
```

Comment:
```suggestion
          "Remaining inflight requests",
```

Reply:
This line could be really long if there are multiple requests, maybe we should have each request on its own line.

Reply:
Hey @connorjclark and @adamraine 
Thanks for the feedback 💪 

From my point of view as long as it's an array that common console/terminal formatters can handle then it should be fine. But maybe you'd like me to add something along the lines of show first 5 requests and after that `...and X more requests` for example? That way it should be pretty neat in case of a lot of remaining inflight requests🤔

Reply:
The `debug` module is going to format the object given using the `inspect` interface. For arrays, that elides the elements.

<img width="499" alt="image" src="https://user-images.githubusercontent.com/4071474/232635553-9a4c166f-65aa-4d50-94cb-da451266da1c.png">


My preference is to join the urls using `, `, and keeping it to a single warn message.

Reply:
Thanks for checking this 💪 

> My preference is to join the urls using `,` , and keeping it to a single warn message.
Do you mean like you posted on your screenshot or `inflightRequestsUrl.join(',')`? I would prefer the first option so they way you posted it on your screenshot 🤔

Reply:
`inflightRequestsUrl.join(', ')`. There shouldn't be on the order of hundreds (or even dozens...) of in flight requests, so I don't think we'd ever even see this log message being elided.

Reply:
If you'd prefer that then sure, but what do you think about just keeping `inflightRequestsUrl` and let the terminals (or wherever the logs are) to handle the formatting? Seems more safe than joining with `,` 🤔  
But if you'd heavily prefer `inflightRequestsUrl.join(', ')` then just let me know and I'll change it to that and we can proceed 😁 
Thank you for the review btw 💪

Reply:
bump
@adamraine & @connorjclark let me know please :D